app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -2,25 +2,37 @@ import React from "react";
 
 import { Formik, Form as FormikForm } from "formik";
 import { Button, Pane } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
+import { Input, Textarea, Select } from "neetoui/formik";
 
-import notesApi from "apis/notes";
+import {
+  NOTES_FORM_VALIDATION_SCHEMA,
+  AVAILABLE_NOTE_TAGS,
+  DUMMY_CONTACTS,
+} from "../constants";
 
-import { NOTES_FORM_VALIDATION_SCHEMA } from "../constants";
+const Form = ({ onClose, note, isEdit, setNotes }) => {
+  const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
 
-const Form = ({ onClose, refetch, note, isEdit }) => {
-  const handleSubmit = async values => {
-    try {
+  const handleSubmit = async value => {
+    await delay(500);
```

Comment:
Is there a need to add a delay here? Could you tell me your reasoning for the same?

Reply:
I was wondering whether causing a synthetic delay would do any better since this operation is going to be async and will take some time.
I will remove it to not create unnecessary complexity.